db-service/test/cqn2sql/__snapshots__/upsert.test.js.snap

```diff
@@ -21,3 +21,14 @@ exports[`upsert test with keys only 1`] = `
   "sql": "INSERT INTO Foo2 (ID) SELECT value->>'$[0]' FROM json_each(?) WHERE true ON CONFLICT(ID) DO NOTHING",
 }
 `;
+
+exports[`upsert test with rows (quoted) 1`] = `
+{
+  "entries": [
+    [
+      "[[1,null,2]]",
+    ],
+  ],
+  "sql": "INSERT INTO """Foo2Quoted""" ("""ID""","""name""","""a""") SELECT value->>'$[0]',value->>'$[1]',value->>'$[2]' FROM json_each(?) WHERE true ON CONFLICT("""ID""") DO UPDATE SET """name""" = excluded."""name""","""a""" = excluded."""a"""",
```

Comment:
the `db-service` tests don't have reserved words configured as these are database specific. Therefor the `order` is not quoted in this test.